src/hooks/core/chat/useChatUtilsHook.ts

```diff
@@ -171,8 +173,8 @@ function useChatUtilsHook(): UseChatUtilsHook {
     });
   };
 
-  const goToContactScreen = () => {
-    navigation.navigate('ContactScreen');
+  const goToContactScreen = ({from}: ContactScreenPayload) => {
+    navigation.navigate('ContactScreen', {from});
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The function `goToContactScreen` now accepts a payload object with a `from` property. This change seems to be intended to provide more context when navigating to the 'ContactScreen'. Make sure that the 'ContactScreen' is set up to handle this additional context.
```diff
- const goToContactScreen = () => {
-   navigation.navigate('ContactScreen');
+ const goToContactScreen = ({from}: ContactScreenPayload) => {
+   navigation.navigate('ContactScreen', {from});
  };
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->